readme.md

```diff
@@ -19,16 +19,20 @@ $ npm install --save is-online
 ```js
 var isOnline = require('is-online');
 
-isOnline(function(err, online) {
-	console.log(online);
-	//=> true
-});
+isOnline()
+	.then(function (online) { 
+		console.log(online);
+		//=> true
+	})
+	.catch(function (err) { 
```

Comment:
Drop the `.catch()`. It's implied that you should deal with that at some point when using promises.